http/src/main/resources/reference.conf

```diff
@@ -1129,6 +1129,12 @@ autoFreeze {
   maxKernelBusyLimit = 24 hours
 }
 
+autoDelete {
+  enableAutoDelete= true
+  autoDeleteAfter = 30 minutes
+  autoDeleteCheckScheduler = 1 minutes
```

Comment:
since realistically in UI, we'll probably give users options to delete after x days...maybe change the interval to be 5 hours or sth? (it does mean to test out auto delete from API, we'll need to wait 5 hrs tho)